src/components/swap-vult/index.tsx

```diff
@@ -183,6 +183,45 @@ const Component: FC = () => {
     }
   };
 
+  const handleUseFullAmount = (ticker: TickerKey) => {
+    if (!loading && isConnected) {
+      let fullAmount = tokens[ticker].balance;
+      
+      // If the token is ETH, we need to reserve some for gas fees
+      if (ticker === TickerKey.ETH) {
+        // Get max network fee using the existing function
+        const estimatedGasFeeEth = getMaxNetworkFee(1) / (values?.[TickerKey.ETH] || 1);
+        
+        // Add a 10% buffer to ensure we have enough for gas fluctuations
+        const gasFeeWithBuffer = estimatedGasFeeEth * 1.1;
+        
+        // If balance is too low, set amount to 0 and show a warning
+        if (fullAmount > gasFeeWithBuffer) {
+          fullAmount -= gasFeeWithBuffer;
+        } else {
+          // Show warning message to the user
+          message.warning(t(constantKeys.INSUFFICIENT_BALANCE) + ". " + 
+                         "Please add more ETH to your wallet for gas fees.");
```

Comment:
Lets store all text in local so we can easily change them in future or support multiple language and also make the code easier to read